bambi/models.py

```diff
@@ -247,9 +248,19 @@ def fit(
 
         Returns
         -------
-        An ArviZ ``InferenceData`` instance if method  ``"mcmc"`` (default).
+        An ArviZ ``InferenceData`` instance if inference_method  ``"mcmc"`` (default).
```

Comment:
Nitpick: Can we also add its InferenceData for "mcmc", "nuts_numpyro" and "nuts_blackjax"?